app/views/applications/show.html.erb

```diff
@@ -0,0 +1,47 @@
+<h1>Application Show Page</h1>
+<p>Name: <%= @application.name %></p>
+<p>Street: <%= @application.street_address %></p>
+<p>City: <%= @application.city %></p>
+<p>State: <%= @application.state %></p>
+<p>Zip: <%= @application.zip_code %></p>
+<p>Description: <%= @application.description %></p>
+<p>Status: <%= @application.application_status %></p>
+
+<% unless @application.pets.empty? %>
+    <h2>Pets You've Applied For:</h2>
+      <% @application.pets.each do |pet| %>
+        <p><%= link_to pet.name, "/pets/#{pet.id}" %></p>
+      <% end %>
+<% end %>
+
+<% if @application.application_status == "In Progress" %>
+    <h1>Add a pet to this application</h1>
+    <%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |form| %>
```

Comment:
I encourage you to replace `local: true` with `data: {turbo: false}` because we're using Rails 7 and Turbo can sometimes interfere with form requests made that redirect back to the same controller action (which this page does!)

Reply:
I see now that you're disabling Turbo below and I'm wondering if you saw different issues with this form that made you bring back `local: true`? Let me know!